src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdaterTest.java

```diff
@@ -52,106 +53,28 @@ public void setUp() {
      *
      */
     @Test
-    public void should_update_case_data_with_exception_record() {
+    public void should_update_case_data_with_exception_record_if_no_warnings() {
         // given
-
-        Address originalCaseAddress = new Address("a", "b", "c", "d", "e", "f", "g");
         Address exceptionRecordAddress = new Address("0", "1", "2", "3", "4", "5", "6");
 
         LocalDateTime caseScannedDate = now();
         LocalDateTime caseDeliveryDate = now();
 
-        SampleCase originalCase = new SampleCase(
-            "legacy-id",
-            "first-name",
-            "last-name",
-            "date-of-birth",
-            "contact-number",
-            "email",
-            originalCaseAddress,
-            singletonList(
-                new Item<>(
-                    new ScannedDocument(
-                        "type_A",
-                        "subtype_A",
-                        new DocumentUrl("file://file_A", "binary_url_A", "file_name_A"),
-                        "control_number_A",
-                        "file_name_AA",
-                        caseScannedDate,
-                        caseDeliveryDate,
-                        "exceptionRecordReference"
-                    )
-                )
-            ),
-            "er-id"
+        List<Item<ScannedDocument>> scannedDocuments = scannedDocuments(
+            caseScannedDate,
+            caseDeliveryDate
```

Comment:
why do we need to pass scanned and delivery date. You can set it as now in the method itself ?